src/main/cpp/DragonVision/DragonVision.cpp

```diff
@@ -220,23 +220,23 @@ std::optional<VisionData> DragonVision::GetVisionDataFromNote(VISION_ELEMENT ele
 		break;
 	case VISION_ELEMENT::NOTE:
 	{
-		bool frontHasDetection = m_dragonCameraMap[RobotElementNames::CAMERA_USAGE::L_INTAKE]->HasTarget();
-		bool backHasDetection = m_dragonCameraMap[RobotElementNames::CAMERA_USAGE::P_INTAKE]->HasTarget();
-		if (!frontHasDetection && !backHasDetection)
+		bool launcherHasDetection = m_dragonCameraMap[RobotElementNames::CAMERA_USAGE::L_INTAKE]->HasTarget();
+		bool placerHasDetection = m_dragonCameraMap[RobotElementNames::CAMERA_USAGE::P_INTAKE]->HasTarget();
+		if (!launcherHasDetection && !placerHasDetection)
 		{
 			return std::nullopt;
 		}
-		else if (frontHasDetection && backHasDetection)
+		else if (launcherHasDetection && placerHasDetection) // we see targets in no cameras
```

Comment:
the comment seems wrong. The if is asking if both launcher and placer see a target

Reply:
Done